Jenkinsfile-e2e

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env groovy
 library 'Simorgh'
 
-// Run latest every 3 hours
-def cron_string = env.BRANCH_NAME == "latest" ? "0 0,6,9,12,15,18,21 * * *" : ""
+// Run latest every 6 hours
+def cron_string = env.BRANCH_NAME == "latest" ? "0 */6 * * *" : ""
```

Comment:
Looks like it didn't used to run exactly every 3 hours, taking a little break before midnight and in the early hours! I assume there wasn't a reason for this?

Reply:
We used to have issues with the tests running at 3am so it was disabled for 3am.

Reply:
We previously had to skip runs at 3am as they would never work